app/_landing_pages/gateway/kong-manager.yaml

```diff
@@ -0,0 +1,147 @@
+metadata:
+  title: Kong Manager
+  product: gateway
+  content_type: landing_page
+  description: Kong Manager is the graphical user interface (GUI) for {{site.base_gateway}}.
+rows:
+  - header:
+      type: h1
+      text: "Kong Manager"
+
+  - columns:
+      - blocks:
+          - type: structured_text
+            config:
+              header:
+                text: "What is Kong Manager?"
+              blocks:
+                - type: text
+                  text: | 
+                    Kong Manager is the graphical user interface (GUI) for {{site.base_gateway}}.
+                    It uses the [Admin API](/api/gateway/admin-ee/) under the hood to administer and control {{site.base_gateway}}. 
+                    If you're running {{site.base_gateway}} on-prem with a database (either in traditional or hybrid mode), you can enable Kong Manager.
+
+                    You can do the following with Kong Manager:
+                - type: unordered_list
+                  items:
+                    - Manage all Workspaces in one place
+                    - Create and manage Routes and Gateway Services
+                    - Activate or deactivate plugins
+                    - Manage Certificates
+                    - Manage users and roles for {{site.base_gateway}}
+                    - Centrally store and easily access Key Sets and Keys
+                    - Manage Vaults
+      - blocks:
+          - type: image
+            config:
+              url: /assets/images/gateway/kong-manager-ui.png
```

Comment:
I wonder if we can resize the screen to a narrower width. Right now it's really small and hard to see the screenshot.

Reply:
Sure! I've fixed this in my latest commit